extensions/control-plane/store/sql/contract-negotiation-store-sql/src/main/java/org/eclipse/edc/connector/store/sql/contractnegotiation/SqlContractNegotiationStoreExtension.java

```diff
@@ -33,9 +34,11 @@
 @Extension(value = "SQL contract negotiation store")
 public class SqlContractNegotiationStoreExtension implements ServiceExtension {
 
-    private static final String DATASOURCE_NAME_SETTING = "edc.datasource.contractnegotiation.name";
     private static final String DEFAULT_DATASOURCE_NAME = "contractnegotiation";
 
+    @Setting(value = "Name of the datasource used to access contract negotiation store", defaultValue = DEFAULT_DATASOURCE_NAME)
```

Comment:
I would describe this as: "Name of the datasource used by SQL-based contract negotiation stores"